packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -2102,18 +2192,14 @@ def _prepare_job_params(self):
             sample.attributes["id"]: sample.media.path for sample in self._boxes_dataset
         }
 
-        # Make job layouts wrt. manifest params
-        # 1 job per task, 1 task for each point label
-        #
-        # Unlike other task types, here we use a grid of RoIs,
-        # so the absolute job size numbers from manifest are multiplied by the job size multiplier.
-        # Then, we add a percent of job tiles for validation, keeping the requested ratio.
-        gt_ratio = self.manifest.validation.val_size / (self.manifest.annotation.job_size or 1)
-        job_size_mult = self.job_size_mult
-
-        job_params: list[self._JobParams] = []
```

Comment:
Why this logic was removed? A multiplier is still used, even though there is no grid logic.

Reply:
reverted logic related to `job_size_mult`